grub2_theme_preview/__main__.py

```diff
@@ -366,7 +366,9 @@ def _grub2_ovmf_tuple():
         '/usr/share/edk2-ovmf/x64/OVMF_CODE.fd',  # Arch Linux and its derivatives
         '/usr/share/OVMF/OVMF_CODE.fd',  # Debian and its derivatives
         '/usr/share/edk2/ovmf/OVMF_CODE.fd',  # Fedora (and its derivatives?)
+        os.environ.get('GRUB2_PREVIEW_OVMF_CANDIDATE', ''), # NixOS and potentially others
     ]
+
```

Comment:
Let's adjust the variable name a bit and only check that very file if given, i.e.:
```python
omvf_image = os.environ.get('G2TP_OVMF_IMAGE')
if omvf_image is not None:
    candidates = [omvf_image]
else:
    candidates = [
        '/usr/share/edk2-ovmf/OVMF_CODE.fd',  # Gentoo and its derivatives
        '/usr/share/edk2-ovmf/x64/OVMF_CODE.fd',  # Arch Linux and its derivatives
        '/usr/share/OVMF/OVMF_CODE.fd',  # Debian and its derivatives
        '/usr/share/edk2/ovmf/OVMF_CODE.fd',  # Fedora (and its derivatives?)
    ]
```
What do you think?

Reply:
LGTM